src/lib/Output.svelte

```diff
@@ -215,7 +215,7 @@
 							class:content={isContent(word)}
 							class:editing={mode === 'edit'}
 							class:connected={connecting.some(([l, w]) => l == i && w == j)}
-							style={`color: ${color_map[i][j] >= 0 ? colors[color_map[i][j]] : 'none'}`}
+							style:color={colors[color_map[i][j]]}
```

Comment:
`none` is not a valid color value in CSS. `unset` should be used instead, but if `color_map[i][j] < 0`, `colors[color_map[i][j]]` will be `undefined` anyway